src/swarm/neo/client/RequestSet.d

```diff
@@ -263,6 +265,8 @@ public final class RequestSet: IRequestSet
             suspended fiber, waiting for I/O to complete, or may have finished.
 
             Params:
+                RequestContext = type of request context. Must be a type that
```

Comment:
I'll not insist to put this in a separate commit.

Reply:
Surely it counts as "Add missing method doc"?